izpack-installer/src/main/java/com/izforge/izpack/installer/gui/InstallerController.java

```diff
@@ -65,12 +71,22 @@ private void run(Runnable action)
         }
         else
         {
+            StringWriter sw = new StringWriter();
+            PrintWriter pw = new PrintWriter(sw);
             try
             {
                 SwingUtilities.invokeAndWait(action);
             }
+            catch (InvocationTargetException e) {
+                Throwable targetException = e.getTargetException();
+                targetException.printStackTrace(pw);
+                logger.info("InvocationTargetException: " + sw);
```

Comment:
Better use the following method for logging in order the logging framework to handle the stacktrace output:
`java.util.logging.Logger#log(Level level, String msg, Throwable thrown)` as so:

```
logger.log(Level.INFO, "Action invocation failed", targetException);
```

Reply:
Done.